### ISSUE TITLE
<Flex bg="blue" /> and <Box bg="blue" /> throw React warnings.

### ISSUE THREAD

**Issue by karland** [jxnblk/rebass#330]
Both `<Flex bg="blue" />` and `<Box bg="blue" />`  work as documented, however, they throw React warnings:

```
Warning: Unknown prop `bg` on <div> tag. Remove this prop from the element. For details, see https://fb.me/react-unknown-prop
    in div (created by Base)
    in Base (created by Box)
    in Box (created by Flex)
    in Flex (created by Styled(Flex))
    in Styled(Flex) (at App.js:24)
    in div (created by t)
    in div (created by t)
    in t (at App.js:18)
    in div (created by Base)
    in Base (created by Styled(Base))
    in Styled(Base) (created by Styled(Styled(Base)))
    in Styled(Styled(Base)) (at App.js:17)
    in App (at Root.js:14)
    in Router (created by BrowserRouter)
    in BrowserRouter (at Root.js:13)
    in Root (at index.js:6)
```

**Comment by costagolub** [comment]
@karland it's not a new issue. Been discussed here #276 #266

**Comment by karland** [comment]
@costagolub Thanks.

**Comment by smith-kyle** [comment]
I'm seeing the warning for
```javascript
<Flex f={1} />
```

IMO this issue should be reopened. #276 and #266 are both closed and the warning hasn't been fixed yet

**Comment by epilande** [comment]
@smith-kyle it looks like `Flex` & `Box` doesn't run through the blacklist.

Alternatively, you can use `Row` & `Column` which should allow you to use the `f` attribute without the unknown props warning.

**Comment by epilande** [comment]
After more investigating, I've found 2 issues:
- `Flex` & `Box` doesn't run through `src/hoc.js` so blacklist isn't filtered out. https://github.com/jxnblk/rebass/pull/334
- `tag-hoc` doesn't clean styled-component props. https://github.com/jxnblk/tag-hoc/pull/2

Anyways, I've submitted PRs that should fix those issues and get rid of those unknown prop warnings for `Flex` & `Box` as well as `Row` & `Column`.
